src/assets/global.scss

```diff
@@ -41,8 +42,37 @@ body {
   text-rendering: optimizeLegibility;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  height: 100%;
+}
+.button-on {
```

Comment:
use scss features; lots of the attributes for button-on and off are the same and therefore redundant